src/CoreWCF.Http/src/CoreWCF.Http.csproj

```diff
@@ -4,6 +4,17 @@
     <AssemblyName>CoreWCF.Http</AssemblyName>
     <PackageId>CoreWCF.Http</PackageId>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <VersionPrefix>0.0.0</VersionPrefix>
+    <VersionSuffix>pre-beta</VersionSuffix>
```

Comment:
We should use one of the recommended suffixes [listed here](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#pre-release-versions), As the api is still changing slightly, as you identified beta isn't appropriate. -alpha seems the best match.